.github/workflows/set_wheel_build_matrix.yml

```diff
@@ -45,6 +48,18 @@ jobs:
               --min-version=3.${{ env.PYTHON3_MIN_VERSION }} \
               --max-version=3.${{ env.PYTHON3_MAX_VERSION }} --range)" >> $GITHUB_OUTPUT
           fi
+      - name: Python version for GPU runners
+        id: pyvergpu
+        run: |
+          if [[ ${{ inputs.event_name }} == 'release' ]]; then
+            echo "python_version_gpu=$(python3 scripts/gen_pyver_matrix.py \
+              --min-version=3.${{ env.PYTHON3_MIN_VERSION }} \
+              --max-version=3.${{ env.PYTHON3_MAX_VERSION }}) --range" >> $GITHUB_OUTPUT
+          else
+            echo "python_version_gpu=$(python3 scripts/gen_pyver_matrix.py \
+              --min-version=3.${{ env.PYTHON3_MIN_VERSION }} \
+              --max-version=3.${{ env.PYTHON3_MAX_VERSION }})" >> $GITHUB_OUTPUT
+          fi
```

Comment:
This is the same as the standard python_version. Right? Could you please explain the need for that?

Reply:
To avoid building GPU devices w/ Python 3.11 on PRs and merges to master.

Reply:
Where is the place we avoid building GPU devices with Python 3.11? I can't see a difference here.

Reply:
I think I found it.

Reply:
No worries.